Apps/W1/Shopify/app/src/Products/Codeunits/ShpfyProductExport.Codeunit.al

```diff
@@ -314,10 +315,14 @@ codeunit 30178 "Shpfy Product Export"
     /// <param name="Item">Parameter of type Record Item.</param>
     /// <param name="ItemUnitofMeasure">Parameter of type Record "Item Unit of Measure".</param>
     local procedure FillInProductVariantData(var ShopifyVariant: Record "Shpfy Variant"; Item: Record Item; ItemUnitofMeasure: Record "Item Unit of Measure")
+    var
+        ItemIsBlockedLbl: Label 'Price is not synchronized because the item is blocked and sales blocked.';
```

Comment:
Naming conflict, that will cause issues with translations.
Don't name both of the ItemIsBlocked labels the same way.